### DISCUSSION TITLE
Can't add more than one field user_meta_data

### DISCUSSION THREAD

**Discussion by ricardosikic** [supabase/supabase#11435] [Questions] [answered]
Hello,
I am doing a user registration through `nodejs`, this was already done in react but only with one field. The problem is that I try to send more than one field in the metadata of the record but it only adds `username` to the profiles table and not the others. However I get user information on the front end and there is the additional metadata that was added.

I'm using `supabase v2.0`

```
const createStripeAccount = async(req, res) => {
  const { username, email, password } = req.body;
  try {
    const account = await stripe.accounts.create({
      type: 'express',
      country: 'US',
      email: email,
      capabilities: {
        card_payments: {requested: true},
        transfers: {requested: true},
      }
    });
    const { id:stripeAccountId } = account;
   
    let newData = null;
    if(stripeAccountId) {
      console.log(typeof stripeAccountId)
      newData = await supabase.auth.signUp(
        {
          email: email,
          password: password,
          options: {
            data: {
              username: username,
              stripe_account_user_id: 'stripeAccountId', <---additional
              is_founder: true, <---additional
              website: 'www.sdsoso.cl' <---additional
            }
          }
        }
      )
    }
    const { data: {user} } = newData;
    console.log(newData)
    return res.json({
      user
    });
    
  } catch(error) {
    console.log('error', error);
  }
}
```
The response from the server is the full metadata sended. But in the table only add username and not the others.

<img width="370" alt="Captura de pantalla 2023-01-03 a la(s) 23 37 07" src="https://user-images.githubusercontent.com/36648562/210474214-0acdfa44-6107-4898-8b1d-b6d870455c8b.png">

<img width="474" alt="Captura de pantalla 2023-01-03 a la(s) 23 39 22" src="https://user-images.githubusercontent.com/36648562/210474467-76af8470-c989-4ed4-8044-145bee5840ac.png">

**Comment by GaryAustin1** [comment, marked as answer]
You must have a trigger function on the auth.users table that copies username meta data to your profiles table.   You will need to add your new data to that sql code.  

Also be careful how you use user metadata as a signed in user can change that data by just calling updateUser thru the API.

**Reply by ricardosikic** [reply]
thanks !
